android/app/build.gradle

```diff
@@ -39,11 +39,11 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "me.tigerkong.hvz_flutter_app"
-        minSdkVersion 16
+        minSdkVersion 18
```

Comment:
This had a conflict with one of our other libraries. This only removes eligibility from 1.4% of all devices world-wide, and upgrades from Android JellyBean 4.1 to 4.3, so I've decided to just go ahead and perform the upgrade. In reality, we should probably be restricting `minSdkVersion` to 21 or 23, but as of right now we're not using such advanced features so this isn't strictly necessary.